### ISSUE TITLE
Mention that the man-pages at man7.org are outdated

### ISSUE THREAD

**Issue by sebastiancarlos** [mkerrisk/man-pages#33]
The website is currently outdated. It shows an outdated version of man-pages ([5.13](https://man7.org/linux/man-pages/changelog.html), released in 2021). Could you add a note saying that on the [homepage](https://man7.org/linux/man-pages/index.html)?

A related [PR](https://github.com/mkerrisk/man-pages/pull/31) wants to mention that this Github mirror is outdated too.

**Comment by Artoria2e5** [comment]
The HTML is pretty good though -- way better than die, which always shows up first on Google. It's not ManKier or the fancy BSD semantic sites, but it's quite readable.

**Comment by sebastiancarlos** [comment]
I agree, but note that other websites like manned.org and ManKier have both good HTML _and_ more up-to-date versions of the man-pages. Compare the versions on https://man7.org/linux/man-pages/man2/pipe.2.html and https://manned.org/pipe.2 or https://www.mankier.com/2/pipe
